test/RemovalUtils.test.ts

```diff
@@ -0,0 +1,81 @@
+import type {
+  RemovalTestHarness,
+  UnpackedRemovalIdV0Struct,
+  UnpackedRemovalIdV0StructOutput,
+} from '../typechain-types/RemovalTestHarness';
+
+import { expect } from '@/test/helpers';
+
+const setupTest = hre.deployments.createFixture(
+  async (
+    hre
+  ): Promise<{
+    removalTestHarness: RemovalTestHarness;
+  }> => {
+    hre.ethernalSync = false;
+    const RemovalTestHarness = await hre.ethers.getContractFactory(
+      'RemovalTestHarness' as unknown as ContractNames
+    );
+    const removalTestHarness =
+      (await RemovalTestHarness.deploy()) as RemovalTestHarness;
+    return {
+      removalTestHarness,
+    };
+  }
+);
+
+const formatRemovalIdData = (
```

Comment:
is this different than the test util you wrote?

Reply:
yeah, this utility just encodes the removal data as bytes to pass it to `createRemovalId`, whereas the other test utility is more involved and includes default convenience values so that we can easily generate a pre-made removal id to use in other places where the focus of the test isn't on the valid creation of the id.